lib/translator/inquiry.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIB_TRANSLATOR_INQUIRY_H
+#define LIB_TRANSLATOR_INQUIRY_H
+
+#include <cstdio>
+
+#include "common.h"
+#include "lib/scsi.h"
+
+#include "absl/types/span.h"
+#include "third_party/spdk/nvme.h"
+
+namespace translator {
+
+StatusCode InquiryToNvme(absl::Span<const uint8_t> scsi_cmd,
+                         nvme::GenericQueueEntryCmd& identify_ns,
+                         nvme::GenericQueueEntryCmd& identify_ctrl,
+                         uint32_t& alloc_len, scsi::LunAddress lun);
+
+StatusCode InquiryToScsi(
+    absl::Span<const uint8_t> raw_cmd, absl::Span<uint8_t> buffer,
+    absl::Span<const nvme::GenericQueueEntryCmd> nvme_cmds);
```

Comment:
Could you add comments describing the pre and post conditions for these methods? In particular, could you briefly describe what is expected of `scsi_cmd` and `raw_cmd` as inputs (their types are not very descriptive). Also, could you briefly describe what the outputs for these functions are? i.e. where can the user expect to find the resulting command after translation.